xbmc/video/ContextMenus.cpp

```diff
@@ -334,7 +369,7 @@ bool CVideoPlayAndQueue::Execute(const std::shared_ptr<CFileItem>& item) const
     const ContentUtils::PlayMode mode = VIDEO_UTILS::IsAutoPlayNextItem(*item)
                                             ? ContentUtils::PlayMode::PLAY_ONLY_THIS
                                             : ContentUtils::PlayMode::PLAY_FROM_HERE;
-    VIDEO_UTILS::PlayItem(item, mode);
+    VIDEO_UTILS::PlayItem(item, "", mode);
```

Comment:
if mode is not "optional" I think it makes sense to move it before the actual "player" optional

Reply:
... but mode is optional.

Reply:
you're right, disregard